docs/TEAM.md

```diff
@@ -56,8 +56,8 @@
 | | [Rakesh](https://github.com/Rakesh-4)              | Software Developer have 3+ years of experience in C#, JavaScript and Angular Framework                                                                                                                                                                                                                                      |
 | | [Samriddhi](https://github.com/sammjainn)          | I'm a student from India. Here to learn and work together!                                                                                                                                                                                                                                                                  |
 | <img src="./images/team/praveen.jpg" width="80px" height="80px">  | [PraveenKumar](https://github.com/praveen2896)     | I'm from India, having 3 years of professional experience in Backend, new to JS,HTML,CSS ,Planning to learn React in future with the help of #twindle.                                                                                                                                                                      |
-| | [Vipin Rawat](https://github.com/aesthytik)        | I'm a frontend Engineer from Punjab,India. 2+ years experience in React Native and React js. I create Logos and graphics for fun.Happy to help :)                                                                                                                                                                           |
-| | [Satyaki Bose](https://github.com/satyaki07)       | I'm a newbie in MERN stack from Kolkata, India with 1+ years of experience. :)                                                                                                                                                                                                                                              |
+|  | [Vipin Rawat](https://github.com/aesthytik)        | I'm a frontend Engineer from Punjab,India. 2+ years experience in React Native and React js. I create Logos and graphics for fun.Happy to help :)                                                                                                                                                                           |
+| <img src="./images/team/satyaki.jpg" width="auto" height="100px"> | [Satyaki Bose](https://github.com/satyaki07)       | I'm a newbie in MERN stack from Kolkata, India with 1+ years of experience. :)                                                                                                                                                                                                                                              |
```

Comment:
Please change the width to 80px and height to auto.

Reply:
made the changes.